src/server/methods.ts

```diff
@@ -108,6 +123,9 @@ const methods: {
       sendTo(ws, { update: [
         ['gameExportData', [JSON.stringify(game.export())]],
       ] });
+
+      // TODO REMOVE ME
+      game.save();
```

Comment:
Should this be removed?

Reply:
Well... yes, but it is a debug feature we might want to keep temporarily? What do you think?

Reply:
Was just checking if you had intended to remove it and just forgot. We can keep it if you want.